manager/controllers/app/plotter_generator.go

```diff
@@ -152,10 +152,10 @@ func (p *PlotterGenerator) AddFlowInfoForAsset(item DataInfo, appContext *app.Fy
 	datasetID := item.Context.DataSetID
 	for _, element := range selection.DataPath {
 		moduleCapability := element.Module.Spec.Capabilities[element.CapabilityIndex]
-		p.Log.Trace().Msg("Adding module for " + moduleCapability.Capability)
+		p.Log.Trace().Msg("Adding module for " + string(moduleCapability.Capability))
```

Comment:
might be cleaner to use Msgf (though it's totally subjective)

Reply:
changed